servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -5889,15 +5889,15 @@ void validate(Object val, String type, String caller) {
     
     Domain createTopLevelDomain(ResourceContext ctx, String domainName, String description,
             String org, Boolean auditEnabled, List<String> adminUsers, String account,
-            int productId, List<String> solutionTemplates, String auditRef) {
+            int productId, List<String> solutionTemplates, String auditRef, String applicationId) {
```

Comment:
Can we please move applicationId argument to be after productId. It'd like to be consistent and have auditRef/caller as last arguments in our APIs.

Reply:
done